src/main/java/org/sagebionetworks/web/client/widget/team/MemberListWidget.java

```diff
@@ -79,10 +78,7 @@ public void onSuccess(TeamMemberPagedResults results) {
 				offset += MEMBER_LIMIT;
 				
 				long numberOfMembers = results.getTotalNumberOfResults();
-				view.setLoadMoreVisibility(offset < numberOfMembers);
-				if (offset < numberOfMembers) {
-					gwtWrapper.scheduleExecution(invokeCheckForInViewAndLoadData, DisplayConstants.DELAY_UNTIL_IN_VIEW);
-				}
+				membersContainer.setIsMore(offset < numberOfMembers);
 				view.addMembers(memberList.getResults(), isAdmin);
```

Comment:
on failure, do not refresh().  use synalert, and membersContainer.setIsMore(false).